docgen/start.js

```diff
@@ -17,6 +17,7 @@ builder({middlewares}, err => {
 
 // then we watch and rebuild
 watch([
+  rootPath('packages/react-instantsearch/src/**/*.js'),
```

Comment:
Do you remember why you had to add this line? The devServer is supposed to handle this part (watch webpack build, rebuild, reload)